lib/oauth_token_verifier/configuration.rb

```diff
@@ -1,7 +1,7 @@
 module OauthTokenVerifier
   class Configuration
     attr_accessor :enabled_providers, :google, :vk, :facebook
-    ProviderSettings = Struct.new(:fields_mapping, :name, :id_field)
+    ProviderSettings = Struct.new(:version, :fields_mapping, :name, :id_field)
 
```

Comment:
This is not going to work, because in the same file below we are initializing the `ProviderSettings` struct for each provider, i.e. 
```
@vk = ProviderSettings.new({
   first_name: :first_name,
   last_name: :last_name
   }, 'vk', 'uid')
```
and this struct will not be correctly initalized, because it has `:version` argument in place of `:fields_mapping`.
Running `bundle exec rspec spec` shows this clearly. You just have to put `version` at the end of the struct for it to work. And then you can move default `5.0` version to the struct initializer.
```
@vk = ProviderSettings.new({
   first_name: :first_name,
   last_name: :last_name
   }, 'vk', 'uid', '5.0')
```

Reply:
Yeah, you're right 👍